fr-FR.xml

```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<localisation>
+    <string id="bd490f4b1d73c0f8057256772b569c69">
+        <![CDATA[rBiblia - un logiciel gratuit pour parcourir les traductions de la Bible]]>
+    </string>
+    <string id="845b40f37ce2fb98e02aabfe5f8ab197">
+        <![CDATA[rBiblia - votre façon de lire la Bible]]>
+    </string>
+    <string id="3383b60e82d66e52e79be4c920e4d527">
+        <![CDATA[Ce site web vous permet de télécharger la dernière version de l'application rBiblia dans laquelle vous pouvez parcourir aisément les traductions de la Bible. Ce logiciel nécessite peu de ressources matérielles et est disponible gratuitement sous environnement Windows. Les principaux avantages de ce logiciel sont:]]>
+    </string>
+    <string id="4d2c4a48e4d0911e22593fd8ba082845">
+        <![CDATA[accès rapide à differentes traductions de la Bible dans différentes langues]]>
+    </string>
+    <string id="7feb865949dead1dfc04fc8435e9cbe7">
+        <![CDATA[recherche à travers les versets en utilisant un certain nombre de phrases]]>
+    </string>
+    <string id="3e162681490fc96808c9995835963a17">
+        <![CDATA[recheche facile d'un verset dans différentes traductions]]>
+    </string>
+    <string id="7cc82a762cdfb2b64db1acc2fbcf8147">
+        <![CDATA[parcours de plusieurs traduction au même moment]]>
+    </string>
+    <string id="daecbc68d7b9dbd9cd4935a444920b9d">
+        <![CDATA[possibilité de changer les couleurs par défaut et la taille de la police]]>
+    </string>
+    <string id="62c167ce6192b9eae37381a5380f74f0">
+        <![CDATA[téléchargement et suppression de traductions simple directement depuis le logiciel]]>
+    </string>
+    <string id="923e37d8b82ea88864eb23f5b58526bf">
+        <![CDATA[copie d'un verset dans le presse-papier]]>
+    </string>
+    <string id="4cbb0fb77750e0bccf2512adf4e04047">
+        <![CDATA[creation de marque-pages complexes pour plusieurs emplacements dans la traduction]]>
```

Comment:
Very basic translation. I need to see the feature in order to improve the translation